examples/openthread_posix_example/README.md

```diff
@@ -0,0 +1,57 @@
+# openthread_posix_example
+
+This example shows the `posix_sockets` example running with OpenThread as network stack.  The usage is exactly the same
```

Comment:
I would prefer the example to run on top of `sock` instead of `posix_sockets` ;-)